src/fs/driver/dvfs_driver.c

```diff
@@ -22,20 +22,20 @@ struct dvfs_driver_head {
 	struct dumb_fs_driver *drv;
 };
 
-ARRAY_SPREAD_DEF(struct dumb_fs_driver *, dumb_drv_tab);
+ARRAY_SPREAD_DEF(const struct dumb_fs_driver *const, dumb_drv_tab);
 
 struct dumb_fs_driver *dumb_fs_driver_find(const char *name) {
-	struct dumb_fs_driver *fs_drv;
+	const struct dumb_fs_driver *fs_drv;
 	array_spread_foreach(fs_drv, dumb_drv_tab) {
 		if (!strcmp(name, fs_drv->name)) {
-			return fs_drv;
+			return (struct dumb_fs_driver *)fs_drv;
```

Comment:
Yes, I'm agree. I didn't such way because I thought that it have been required a lot of changes.
FIx it in [44440ed65d5bbb5e0622d1e092ac479345230be1](https://github.com/embox/embox/commit/44440ed65d5bbb5e0622d1e092ac479345230be1)